CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 
 This project follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.22.1] - 2021-10-4
```

Comment:
I can't see it on this commit, but does the bottom of the file have the link bit? `[v0.22.1]: github..etc..`

Reply:
Good catch!